docs/versioning.md

```diff
@@ -0,0 +1,32 @@
+Versioning Strategy
+===================
+
+# Version Format
+
+The Broker version is stored in the broker's RPM spec file,
+[ansible-service-broker.spec](../ansible-service-broker.spec), and is in the
+form of MAJOR.MINOR.PATCH incremented by the following rules:
+
+* MAJOR version when incompatible API changes are made.
+* MINOR version when a new version of
+  [openshift/origin](https://github.com/openshift/origin) is being targeted.
+* PATCH version when tagged via `tito tag`.
+
+NOTE: The first official release of the broker,
+[`1.0.x`](https://github.com/openshift/ansible-service-broker/tree/ansible-service-broker-1.0.1-1),
+targeted OpenShift Origin [release
+v3.7.0](https://github.com/openshift/origin/releases/tag/v3.7.0). Following the
+versioning rules, the version was bumped to `1.1.x` for OpenShift Origin [release
+v3.9.0](https://github.com/openshift/origin/releases/tag/v3.9.0) and again to
```

Comment:
This link does not appear to be working for me.

Reply:
Yeah. This is because v3.9.0 is not yet released. I could point it to the release-3.9 branch....not sure the best way to handle that.

Reply:
Oh duh, that was silly. I think that pointing to the branch, for now, makes sense. Should we just remember to come back and update that?

Reply:
We should. https://trello.com/c/iT8k66Zl/700-update-broker-readmemd-when-v390-is-released created for it.